clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/InfinispanSessionManager.java

```diff
@@ -129,7 +130,7 @@ public void accept(ImmutableSession session) {
 
     @Override
     public void start() {
-        this.executor = new DefaultExecutorService(this.getClass(), ExecutorServiceFactory.CACHED_THREAD);
+        this.executor = Executors.newCachedThreadPool(new DefaultThreadFactory(this.getClass()));
```

Comment:
@pferraro Will this result in a meaningfully named thread? If not is this a regression or was it not meaningfully named before?

Reply:
This does not change the way threads are named.  The same thread factory is still used.